Resources/Prototypes/_CM14/Entities/Objects/Medical/healing.yml

```diff
@@ -0,0 +1,98 @@
+- type: entity
+  parent: BaseItem
+  id: CMBaseHealingItem
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: _CM14/Objects//Medical/medical.rsi
+  - type: Item
+    size: Small
+    sprite: _CM14/Objects//Medical/medical.rsi
+    heldPrefix: ointment
+  # Inherited
+  - type: StaticPrice
+    price: 0
+
+- type: entity
+  name: ointment
+  description: Used to treat those nasty burns. Less effective on caustic burns.
+  parent: CMBaseHealingItem
+  id: CMOintment  
+  suffix: Full
+  components:
+  - type: Tag
+    tags:
+    - CMOintment
+  - type: Sprite
+    state: ointment
+  - type: Item
+    heldPrefix: ointment
+  - type: Healing
+    damageContainers:
+      - Biological
+    damage:
+      types:
+        Heat: -5
+        Cold: -5
+        Shock: -5
+        Caustic: -1.5
+    healingBeginSound:
+      path: "/Audio/Items/Medical/ointment_begin.ogg"
```

Comment:
i believe path should not start with slash.

Reply:
For audio paths  sspecifically they should, since they need to be rooted